public/index.html

```diff
@@ -15,6 +15,12 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
```

Comment:
think this should be a link to our dotgov.min.css? If thats the case  if the bootstrap stuff doesnt exist in our project we know to update it.